docker/README.md

```diff
@@ -1,44 +1,30 @@
-# 🪐 Docker Image and Container for Validate Tool
+# 🚢 Dockerfile
 
-The docker image of Validate Tool provides an easy way to deploy the Validate Tool, without having to 
-worry about the Java/JDK versions installed and operating systems supported. Also, this helps to 
-easily use the Validate Tool in other execution environments such as docker compose and Amazon ECS.
+There's one Dockerfile here that can be used to make images for development and production.
 
-The only prerequisite to use the dockerized Validate Tool is the availability of docker software in the host computer.
-At the time of writing this, the Validate Tool was tested with Docker version 20.10.12.
+To build an image, run:
 
-## 🏃 Steps to build the docker image of the Validate Tool
+    docker image build --build-arg tar_file=URL --tag [OWNER/]registry-api-service:TAG .
```

Comment:
Do we have to mention about the way to get this URL?

Reply:
Maybe, it would be safer. Is it a tar.gz ? You can give an example, others will easily extrapolate from the example to point to other versions.

Also the tag should be `validate`, not `registry-api-service`

Reply:
@ramesh-maddegoda @tloubrieu-jpl updated and works

Reply:
@ramesh-maddegoda @tloubrieu-jpl @nutjob4life 

do we need to support URLs or since this is part of the CICD mean we can just use local files?

Using `COPY` seems to fix the issue with the docker container not building, but then it cannot accept URLs.

The issue I was seeing using `ADD` is because for some reason, this also includes a hidden step where it decompresses the tar.gz into a folder. So we would need to add additional logic to figure all that out, or we could just assume local file path and be done with it.

Referring to comments below, the ADD [documentation](https://docs.docker.com/engine/reference/builder/#add) states that:
> If is a local tar archive in a recognized compression format (identity, gzip, bzip2 or xz) then it is unpacked as a directory. Resources from remote URLs are not decompressed.

Reply:
I think `ADD` has too much magic 😇